doc/source/reference/random/legacy.rst

```diff
@@ -0,0 +1,113 @@
+.. _legacy:
+
+Legacy Random Generation
+------------------------
+The `~mtrand.RandomState` provides access to
+legacy generators.  These all depend on Box-Muller normals or
+inverse CDF exponentials or gammas. This class should only be used
+if it is essential to have randoms that are identical to what
+would have been produced by NumPy.
+
+`~mtrand.RandomState` adds additional information
+to the state which is required when using Box-Muller normals since these
+are produced in pairs. It is important to use
+`~mtrand.RandomState.get_state`
+when accessing the state so that these extra values are saved.
+
+.. code-block:: python
```

Comment:
This blocks is not longer accurate.  It only made sense for a generator installed next to NumPy. [TODO] Need clean up.

Reply:
Fixed in a PR